src/common/webrtc/api.rs

```diff
@@ -0,0 +1,463 @@
+#![allow(dead_code)]
```

Comment:
This line made sense while building out the POC, but I imagine it should be removed before this is merged in, yeah?